site/docs/v1/tech/messengers/index.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Messengers
+description: Adding and Configuring Messengers
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Messengers
+
+Currently, messengers allow a user to send a message within FusionAuth. They can be customized using link:/docs/v1/tech/themes/[themes].
+
+== Setup a Messenger
+
+In FusionAuth, you can set up three types of messengers
+
+- link:twilio-messenger[Setup a Twilio Messenger]
+- link:kafka-messenger[Setup a Kafka Messenger]
+- link:generic-messenger[Setup a Generic Messenger]
+
+== Create a Messenger Template
+
+Message templates are a powerful way to customize messengers and the communication contained within. FusionAuth uses FreeMarker for all templating.
+
+- link:message-templates[Manage Message Templates]
+
```

Comment:
Would add navigation here. "To configure a message template, navigate to [breadcrumb]#Customizations -> Message Templates".